setup.py

```diff
@@ -37,20 +37,20 @@
 """
 
 import glob
-import os.path
+import os
 import subprocess
 import sys
 
 from distutils.spawn import find_executable
 from pkg_resources import get_build_platform
-from setuptools import setup, find_packages
+from setuptools import setup
 from setuptools.extension import Extension
 
 # Execute DAKOTA to get version.
 try:
```

Comment:
I get your comments (also kind of get where they originated). 
This try: except was in the original version coming from the pyDAKOTA project,
I guess your point here is that this call to dakota is going to fail with some message anyway and we should let that happen. I have nothing against that.